### PR TITLE
bump emsim to 1.0.1

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

**Comment by mgeplf** [comment]
Haven't done this for a while, any feedback is welcome

File: deploy/environments/applications.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ spack:
     - brion@3.3.1 +python
     - circuit-build
     - connectome-tools@0.3.3
-    - emsim ^brion@3.1.0
+    - emsim@1.0.1
```
Comment (matz-e) [inline deploy/environments/applications.yaml]:
```suggestion
    - emsim
```
Leave the version number off, it'll always deploy just the latest version.

File: var/spack/repos/builtin/packages/emsim/package.py [hunk 2]
```diff
@@ -14,16 +14,11 @@ class Emsim(CMakePackage):
 
     generator = 'Ninja'
 
+    version('develop')
     version('1.0.0', tag='v1.0.0', submodules=True)
+    version('1.0.1', tag='v1.0.1')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/emsim/package.py:18]: Submodules not needed any more?

File: deploy/environments/applications.yaml (hunk 1 above)
Comment (mgeplf) [reply deploy/environments/applications.yaml]: that's preferable? If so, cool w/ me

File: var/spack/repos/builtin/packages/emsim/package.py (hunk 2 above)
Comment (mgeplf) [reply var/spack/repos/builtin/packages/emsim/package.py:18]: yup, they are purged

**Review by matz-e — DISMISSED** [review]

**Comment by mgeplf** [comment]
I don' thave merge permission; especially for 'BlueBrain5 Deployment Test Build' failing when it's required.

**Comment by matz-e** [comment]
https://bbpcode.epfl.ch/ci/blue/organizations/jenkins/hpc.spack-deployment.pull-requests/detail/hpc.spack-deployment.pull-requests/1498/tests

Shows four failed compilations of `emsim`… I don't see any 1.0.1 tag in Gerrit…

**Comment by mgeplf** [comment]
I missed the most important part of this: repo is on github now :(

**Comment by mgeplf** [comment]
Fixed URL, removed old version, since layout/dependency management has changed so much.

**Review by matz-e — APPROVED** [review]
All green, it's a go!

**Pull request merged**